### ISSUE TITLE
Warning when installing Pithos on Ubuntu 17.04

### ISSUE THREAD

**Issue by behrangsa** [pithos/pithos#460]
Warning message:

```
/usr/lib/python3/dist-packages/pithos/util.py:66: SyntaxWarning: name '_current_collection' is assigned to before global declaration
  global _current_collection
```

Steps to reproduce:

```
$ sudo add-apt-repository ppa:pithos/ppa
 
 More info: https://launchpad.net/~pithos/+archive/ubuntu/ppa
Press [ENTER] to continue or ctrl-c to cancel adding it

gpg: keybox '/tmp/tmpv8lh47sk/pubring.gpg' created
gpg: /tmp/tmpv8lh47sk/trustdb.gpg: trustdb created
gpg: key 83549DA35C8B1281: public key "Launchpad PPA for Pithos Developers" imported
gpg: Total number processed: 1
gpg:               imported: 1
OK


$ sudo apt update
Hit:1 http://au.archive.ubuntu.com/ubuntu zesty InRelease
Get:2 http://au.archive.ubuntu.com/ubuntu zesty-updates InRelease [89.2 kB]                                                                            
Hit:3 http://au.archive.ubuntu.com/ubuntu zesty-backports InRelease                                                                                                             
Get:4 http://au.archive.ubuntu.com/ubuntu zesty-updates/main amd64 DEP-11 Metadata [208 B]                                            
Get:5 http://au.archive.ubuntu.com/ubuntu zesty-updates/universe amd64 DEP-11 Metadata [2,188 B]                                                                                    
Get:6 http://security.ubuntu.com/ubuntu zesty-security InRelease [89.2 kB]                                                                           
Get:7 http://ppa.launchpad.net/pithos/ppa/ubuntu zesty InRelease [15.4 kB]     
Hit:8 http://repository.spotify.com stable InRelease                                        
Hit:9 https://deb.opera.com/opera-stable stable InRelease                                        
Get:10 http://ppa.launchpad.net/pithos/ppa/ubuntu zesty/main amd64 Packages [668 B]
Get:11 http://ppa.launchpad.net/pithos/ppa/ubuntu zesty/main i386 Packages [668 B]
Get:12 http://ppa.launchpad.net/pithos/ppa/ubuntu zesty/main Translation-en [492 B]
Fetched 198 kB in 1s (99.9 kB/s)         
Reading package lists... Done
Building dependency tree       
Reading state information... Done
All packages are up to date.

$ sudo apt install --install-recommends pithos
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  appstream-util
The following NEW packages will be installed:
  appstream-util pithos
0 to upgrade, 2 to newly install, 0 to remove and 0 not to upgrade.
Need to get 132 kB of archives.
After this operation, 691 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://au.archive.ubuntu.com/ubuntu zesty/universe amd64 appstream-util amd64 0.6.9-1 [37.3 kB]
Get:2 http://ppa.launchpad.net/pithos/ppa/ubuntu zesty/main amd64 pithos all 1.3.0-1ppa1~zesty1 [95.0 kB]
Fetched 132 kB in 1s (82.7 kB/s)                    
Selecting previously unselected package appstream-util.
(Reading database ... 205730 files and directories currently installed.)
Preparing to unpack .../appstream-util_0.6.9-1_amd64.deb ...
Unpacking appstream-util (0.6.9-1) ...
Selecting previously unselected package pithos.
Preparing to unpack .../pithos_1.3.0-1ppa1~zesty1_all.deb ...
Unpacking pithos (1.3.0-1ppa1~zesty1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Processing triggers for desktop-file-utils (0.23-1ubuntu2) ...
Processing triggers for libglib2.0-0:amd64 (2.52.0-1) ...
Processing triggers for man-db (2.7.6.1-2) ...
Processing triggers for gnome-menus (3.13.3-6ubuntu5) ...
Processing triggers for hicolor-icon-theme (0.15-1) ...
Setting up appstream-util (0.6.9-1) ...
Setting up pithos (1.3.0-1ppa1~zesty1) ...
/usr/lib/python3/dist-packages/pithos/util.py:66: SyntaxWarning: name '_current_collection' is assigned to before global declaration
  global _current_collection

```

**Comment by JasonLG1979** [comment]
That was fixed in 1.3.1 just waiting for @TingPing to update the PPA.

**Comment by TingPing** [comment]
1.3.1 pushed and being built atm.
